instrumentation-api-semconv/src/test/java/io/opentelemetry/instrumentation/api/instrumenter/http/HttpClientMetricsTest.java

```diff
@@ -40,24 +38,24 @@ void collectsMetrics() {
 
     Attributes requestAttributes =
         Attributes.builder()
-            .put("http.method", "GET")
-            .put("http.url", "https://localhost:1234/")
-            .put("http.target", "/")
-            .put("http.scheme", "https")
-            .put("net.peer.name", "localhost")
-            .put("net.peer.port", 1234)
-            .put("http.request_content_length", 100)
+            .put(SemanticAttributes.HTTP_REQUEST_METHOD, "GET")
+            .put(SemanticAttributes.URL_FULL, "https://localhost:1234/")
+            .put(SemanticAttributes.URL_PATH, "/")
+            .put(SemanticAttributes.URL_QUERY, "q=a")
+            .put(SemanticAttributes.SERVER_ADDRESS, "localhost")
+            .put(SemanticAttributes.SERVER_PORT, 1234)
             .build();
 
     Attributes responseAttributes =
         Attributes.builder()
-            .put("http.status_code", 200)
-            .put("http.response_content_length", 200)
-            .put(SemanticAttributes.NET_PROTOCOL_NAME, "http")
-            .put(SemanticAttributes.NET_PROTOCOL_VERSION, "2.0")
-            .put("net.sock.peer.addr", "1.2.3.4")
-            .put("net.sock.peer.name", "somehost20")
-            .put("net.sock.peer.port", 8080)
+            .put(SemanticAttributes.HTTP_RESPONSE_STATUS_CODE, 200)
```

Comment:
What do we think about static imports for these to improve test readability?

Reply:
Honestly I don't like them; static imports made this PR harder to write. If we ever decide to change things in the future again, I think I'd prefer to not have static imports here.

Reply:
Might as well add the whole package name in there as well then...make it even LONGER. 🤪

Reply:
🪄 ENTERPRISE 🪄  you mean?